src/soho/dropdown/soho-dropdown.component.ts

```diff
@@ -487,28 +531,60 @@ export class SohoDropDownComponent implements AfterViewInit, AfterViewChecked, O
    */
   public selectValue(value: any): void {
     if (this.dropdown) {
-      // @todo lookup model value!
       this.ngZone.runOutsideAngular(() => {
         this.dropdown.selectValue(value);
       });
     }
   }
 }
 
+/**
+ * Provides a 'wrapper' around the {ControlValueAccessor} added by
+ * angular when handling `select` elements.
+ *
+ * This class allows the {SohoDropDownComponent} to interoperate with
+ * the {ControlValueAccessor}.  Specifically, providing access to the
+ * onChange function, which we must call when the value of the dropdown
+ * is modified.
+ *
+ * @class IdsDropDownControlValueAccessor
```

Comment:
Should be SohoDropDownControlValueAccessorDelegator?

Reply:
Yes - local version has that ;-)